src/StackInterface.php

```diff
@@ -1,11 +1,11 @@
 <?php
 
-namespace Psr\Http\Middleware;
+namespace Idealo\Middleware;
 
-use Psr\Http\Message\RequestInterface;
-use Psr\Http\Message\ResponseInterface;
+use Interop\Http\ServerMiddleware\DelegateInterface;
+use Interop\Http\ServerMiddleware\MiddlewareInterface;
 
-interface StackInterface
+interface StackInterface extends DelegateInterface
 {
     /**
      * Return an instance with the specified middleware added to the stack.
```

Comment:
if that Interop guys don't use anything but ```process(ServerRequestInterface $request)```. I honestly don't see any usecase for ```withMiddleware(MiddlewareInterface $middleware)``` to proccess a Stack. May be by adding middlewares on runtime, but what could be good for?